gapis/api/gles/api/draw_commands.api

```diff
@@ -374,6 +374,33 @@ sub void WriteFramebufferAttachment(FramebufferAttachment a) {
   }
 }
 
+// DependencyReadTexture read all relevant fields of texture to create dependencies
+sub void DependencyReadTexture(ref!Texture texture) {
```

Comment:
We should mark this as @spy_disabled, since there is nothing that I can see in this function that needs to be done during trace. It will save us some performance.

Reply:
Can we also mark `DrawCommandDependencies()` as @spy_disabled?
It is only read/writes for dependencies.

Reply:
Anything that doesn't change what is actually captured by Gapii can be spy_disabled.